src/applications/personalization/profile/components/connected-apps/actions/index.js

```diff
@@ -93,3 +98,15 @@ export function dismissDeletedAppAlert(appId) {
   return async dispatch =>
     dispatch({ type: DELETED_APP_ALERT_DISMISSED, appId });
 }
+
+export function getPrivacyPolicy(appTitle) {
+  return async dispatch => {
+    await apiRequest(`${appsUrl}/${appTitle}`, { apiVersion: 'services' })
```

Comment:
Does this API actually work from the local env?

Reply:
Yes. The apps api service is not at the root api route and is versioned separately. The endpoint structure looks like this: `/services/apps/v0/directory/`, so here I'm overwriting the api version.